sdk/dotnet/Pulumi/Core/Urn.cs

```diff
@@ -8,17 +8,17 @@ namespace Pulumi
     /// An automatically generated logical URN, used to stably identify resources. These are created
     /// automatically by Pulumi to identify resources.  They cannot be manually constructed.
     /// </summary>
-    internal static class Urn
+    public static class Urn
     {
         /// <summary>
         /// Computes a URN from the combination of a resource name, resource type, optional parent,
         /// optional project and optional stack.
         /// </summary>
         /// <returns></returns>
-        internal static Output<string> Create(
+        public static Output<string> Create(
```

Comment:
Exposed this method to use it in one of the tests

Reply:
Any other way to consume this from the test without exposing the method?

Reply:
Our integration tests are just Pulumi programs, so I don't think so... FWIW, the same method is exposed in the Node.js SDK too.